RELEASE_NOTES

```diff
@@ -1,3 +1,15 @@
+========================================================================
+Amazon CloudWatch Agent 1.300043.0 (2024-08-06)
+========================================================================
+Bug Fixes:
+* [Metrics/Plugin] Drop original metrics support added for statsd, collectd and ethtool metrics
+* [Logs/Windows Event] Add windows event log service restart detection and resubscribe
+
+Enhancements:
+* [Metrics/JMX] Add cumulative to delta conversion for JMX metrics
+* [ApplicationSignals] Rename and group application signals attributes
+* Upgrade OTEL Contrib to v0.103.0
```

Comment:
```suggestion
* Upgrade OTEL dependencies to v0.103.0
```

Reply:
I followed the same pattern as here -  https://github.com/aws/amazon-cloudwatch-agent/blob/main/RELEASE_NOTES#L57

Reply:
Yeah I'm a bit surprised that we worded it that way before.